src/components/HomepageFeatures/index.tsx

```diff
@@ -1,99 +1,107 @@
-  //   to: '/docs/general/testing',
-  //   description: 'A summation of libraries used by the developers of the city of Amsterdam',
-  // },
   {
+    image: TestingIcon,
+    title: 'Testing',
+    to: '/docs/general/testing',
+    description: 'A summation of libraries used by the developers of the city of Amsterdam',
+  },
+  {
+    image: GitHubIcon,
     title: 'Using Git',
     to: '/docs/general/using-git',
     description: 'Our policy how to use Git',
   },
   {
+    image: ThirdPartyIcon,
     title: 'Third party dependencies in general',
     to: '/docs/general/third-party-dependencies',
     description: 'A guideline on how to choose',
   },
   {
+    image: ThirdPartyIcon,
     title: 'Frontend - Third party dependencies',
     to: '/docs/frontend/third-party-dependencies',
     description:
       ' A list of recommended packages and set guidelines for choosing a package which is not on the list.',
   },
   {
+    image: ReactIcon,
     title: 'Frontend - Languages and Frameworks (Not yet formatted to meet the required standard.)',
     to: '/docs/frontend/languages-and-frameworks',
     description:
       'For all frontend projects within de Municipality of Amsterdam we choose to work with React and its ecosystem.',
   },
   {
+    image: SharedIcon,
     title: 'Frontend - Shared components (Not yet formatted to meet the required standard.)',
     to: '/docs/frontend/shared-components',
     description:
       'A list of components that our being used by the developers of the city of Amsterdam',
   },
-  {
-    title: 'Frontend - Testing (Not yet formatted to meet the required standard.)',
-    to: '/docs/frontend/testing',
-    description: 'A guideline to testing for frontenders.',
-  },
 ];
 
 export default function HomepageFeatures(): JSX.Element {
-  const list = featureList.map((listItem) => (
-    <div key={uuidv4()}>
-      <UnorderedList>
-        <UnorderedList.Item>
-          <Heading size="level-5">
-            <Link to={listItem.to}>{listItem.title}</Link>
-          </Heading>
-          <Paragraph>{listItem.description}</Paragraph>
-        </UnorderedList.Item>
-      </UnorderedList>
-    </div>
-  ));
-
   return (
     <>
-      <Grid.Cell span={12}>
-        <Heading size="level-4">Current Documented Standards</Heading>
-      </Grid.Cell>
-      <Grid.Cell span={12}>
-        <div>{list}</div>
-      </Grid.Cell>
+      {featureList.map((listItem) => (
+        <Grid.Cell key={uuidv4()} span={{ narrow: 8, medium: 6, wide: 4 }}>
+          <Card>
+            {listItem.image &&
+              React.createElement(listItem.image, {
+                width: 62,
+                height: 62,
+                preserveAspectRatio: 'xMinYMin',
+                className:
+                  listItem.title === 'Using Git' ? 'dark-mode-github-icon' : 'ams-card__image',
+              })}
+            <Heading size="level-4">
+              {listItem.title}
+              <Card.Link href={listItem.to} />
```

Comment:
In design-system, the Card example wraps the Heading text in a `Card.Link`, is there a reason for not doing that here?

design-system:
```tsx
<Card>
  <Heading size="level-4">
    <Card.Link href="/">
      Kennisgevingen en bekendmakingen
    </Card.Link>
  </Heading>
  <Paragraph>
    Kennisgevingen en bekendmakingen van de gemeente Amsterdam van de laatste 8 weken.
  </Paragraph>
</Card>
```

Otherwise I would just wrap the <Heading> and <Paragraph> in an anchor, that seems to work. Also link titles should be color `#004699` and on hover, underlined. In dark mode the color is white.

Reply:
The color was added automatically when I used `Link` from the design system so I guess that's the correct color. 

The `paragraph` becomes also a link when you wrap the `heading` in a `card.link`, but doesn't get the hover status (underlined). Kept it like that since the design system thinks that's the way forward.